stations.csv

```diff
@@ -26825,3 +26825,6 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
 28956;Bari Torre Quetta;bari-torre-quetta;8311004;;41.111566;16.91585;22157;f;IT;f;Europe/Rome;t;;;f;;f;8311004;f;;f;;f;8311004;t;S11004;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 28957;Coppet;coppet;;;46.317652;6.187923;;f;CH;f;Europe/Zurich;f;;;f;;f;8501023;f;;f;;f;8501023;t;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 28958;Malpensa Aeroporto T2;malpensa-aeroporto-t2;8301146;;45.646821;8.723376;20530;f;IT;f;Europe/Rome;f;;;f;;f;;f;;f;;f;8301146;t;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
+28959;Courmayeur;courmayeur;8313836;;45.79659;6.96893;;f;IT;f;Europe/Rome;t;;;f;;f;;f;;f;;f;8313836;t;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
+28960;Madonna di Campiglio;madonna-di-campiglio;8313837;;46.22671;10.82582;;f;IT;f;Europe/Rome;t;;;f;;f;;f;;f;;f;8313837;t;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
+28961;Cortina d'Ampezzo;cortina-dampezzo;8313835;;46.5369;12.13903;;f;IT;f;Europe/Rome;t;;;f;;f;;f;;f;;f;8313835;t;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
```

Comment:
Fixed. Should we add a unit test for this? I noticed other stations are using `'`.